apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -211,6 +218,87 @@ export class TeacherFeedback extends Component {
     return feedbackUnchanged;
   };
 
+  getFriendlyDate = feedbackSeen => {
+    const today = new Date();
+    const dateFeedbackSeen = new Date(feedbackSeen);
+
+    //javascript months begin counting at 0
+    const [todayM, todayD, todayY] = [
+      today.getMonth() + 1,
+      today.getDate(),
+      today.getFullYear()
+    ];
+
+    const [m, d, y] = [
+      dateFeedbackSeen.getMonth() + 1,
+      dateFeedbackSeen.getDate(),
+      dateFeedbackSeen.getFullYear()
+    ];
+    if (todayM === m && todayY === y) {
+      if (todayD === d) {
+        return 'today';
+      } else if (todayD === d + 1) {
+        return 'yesterday';
+      }
+    } else {
+      const feedbackYear = (y + '').substr(2, 2);
+      return `${m}/${d}/${feedbackYear}`;
```

Comment:
This will also need to be internationalized. I'd recommend going ahead and using moment.js to print this string as you were originally intending. That way users with mm/dd/yy and users with dd/mm/yy will both be supported as well as users who don't use Arabic numerals for dates. Note: The two digit year isn't a hard requirement.

Reply:
TimeAgo uses moment.js `.fromNow()` method, which will return text such as "two days ago" instead of 10/6/20. Is that acceptable or does it deviate too much from the original spec?

Reply:
Updated to using moment.js `moment(dateFeedbackSeen).format('l')`. Screenshots in PR description are up to date.